src/components/WebexMember/WebexMember.jsx

```diff
@@ -10,23 +10,38 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
  *
  * @param {string} props.personID  ID of the person for which to display avatar
  * @param {boolean} props.displayStatus  Whether or not to display the user's status
+ * @param {string} props.destinationID  ID of the destination for which to get members
+ * @param {string} props.destinationType Type of destination of the membership roster
  *
  * @returns {object} JSX of the component
  */
-export default function WebexMember({personID, displayStatus}) {
+export default function WebexMember({
+  personID,
+  displayStatus,
+  destinationID,
+  destinationType,
+}) {
   const {firstName, lastName} = usePerson(personID);
+  const memberships = useMemberships(destinationID, destinationType);
```

Comment:
actually, `members`, not `memberships`. The hook should also be renamed to `useMembers`, but maybe not part of this PR.